app/controllers/katello/api/v2/content_views_controller.rb

```diff
@@ -175,6 +188,46 @@ def remove
       respond_for_async :resource => task
     end
 
+    api :PUT, "/content_views/:id/bulk_remove", N_("Bulk remove versions from a content view and reassign systems and keys")
```

Comment:
This api's name 'bulk_remove' isn't really indicative of the resulting actions it takes as it doesn't perform a 'removal' but rather a 'deletion' of versions. 

After discussing with @jlsherrill, this should likely be updated to better reflect its actions.